docs/UserGuide.md

```diff
@@ -120,13 +120,9 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/looking for HDB`
 * `add n/Betsy Crowe t/condo e/betsycrowe@example.com a/Flatbush Avenue, block 81, #02-02 p/1234567`
 
+Visual example of correct output:
```

Comment:
Please input on how you feel about me labelling it as `Visual example of correct output:`. Is there a better phrasing of this?

Reply:
lgtm